### PR TITLE
snake-first-attempt

### PR DESCRIPTION
**Pull request by kostya-konev**
- [DEMO LINK](https://kostya-konev.github.io/layout_snake/)
- [TEST REPORT LINK](https://kostya-konev.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  background-color: rgb(51, 0, 0);
+  order: 2;
+}
+
+.box:last-child {
+  background-color: rgb(0, 0, 0);
+  order: 1;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:53]: do you really need (min-width: 600px) here?

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (kostya-konev) [reply src/style.css:53]: No, sorry.
That's my mistake

Comment (kostya-konev) [reply src/style.css]: Я исправил

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job

**Pull request closed without merging**